README.md

```diff
@@ -21,16 +21,15 @@ At some point in the future they might merge, or one might depend on the other.
 Right now here are the differences:
 
  - FiniteDifferences.jl supports basically any type, where as FiniteDiff.jl supports only array-ish types
- - FiniteDifferences.jl supports higher order approximation
+ - FiniteDifferences.jl supports higher order approximation and step size adaptation
  - FiniteDiff.jl is carefully optimized to minimize allocations
```

Comment:
We can delete that now?
Or change it to say "both are..."

Do we have a "both are" section?
Maybe we should.
"Both are carefully optimized to minimize allocations, both are expertly designed to give high accuracy finite differencing based derivative estimates."

Reply:
Hmm, perhaps FiniteDiff.jl still has the advantage on the allocation front by supporting in-place computation and this package the advantage on the robustness/accuracy front by supporting high-order methods and step size adaptation. Maybe change the sentence to "FiniteDiff.jl supports in-place computation"?